data/fields/heritage.json

```diff
@@ -0,0 +1,6 @@
+{
+    "key": "heritage",
+    "type": "number",
+    "minValue": 1,
+    "label": "Heritage Registration Level"
```

Comment:
I wonder if this will make any sense to someone who’s mapping heritage buildings but hasn’t encountered `admin_level=*` yet (since boundaries are hidden by default). Some heritage registries categorize listed structures into what the user might confuse with “registration levels”. But this field is about the heritage organization, not the structure. Since the `admin_level=*` field is called “Admin Level”, could we call this field “Heritage Register Admin Level”?